core/src/Factory.sol

```diff
@@ -40,60 +39,95 @@ contract Factory {
 
     event EnrollCourier(uint32 indexed id, address indexed wallet, uint16 cut);
 
+    event SetMarketConfig(IUniswapV3Pool indexed pool, MarketConfig config);
+
+    // This `Factory` can create a `Market` for any Uniswap V3 pool
     struct Market {
+        // The `Lender` of `token0` in the Uniswap pool
         Lender lender0;
+        // The `Lender` of `token1` in the Uniswap pool
         Lender lender1;
+        // The implementation to which all `Borrower` clones will point
         Borrower borrowerImplementation;
     }
 
+    // Each `Market` has a set of borrowing `Parameters` to help manage risk
     struct Parameters {
+        // The amount of Ether a `Borrower` must hold in order to borrow assets
         uint208 ante;
+        // To avoid liquidation, a `Borrower` must be solvent at TWAP * e^{± nSigma * IV}
         uint8 nSigma;
+        // Borrowing is paused when the manipulation metric > threshold; this scales the threshold up/down
         uint8 manipulationThresholdDivisor;
+        // The time at which borrowing can resume
         uint32 pausedUntilTime;
     }
 
-    struct Courier {
-        address wallet;
-        uint16 cut;
-    }
-
+    // The set of all governable `Market` properties
```

Comment:
Why does this keep getting moved??

Reply:
In the previous PR it was sitting amongst the state variables, so I moved it up. Here it's just being shifted down because I added a new struct